user/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from .views import SignUp
+
+urlpatterns= [
+        path('signup/',SignUp.as_view()),
```

Comment:
앞서 언급한 내용과 상응하는 부분입니다.
```suggestion
        path('/signup',SignUp.as_view()),
```